ingestion/src/metadata/ingestion/source/storage/s3/metadata.py

```diff
@@ -262,34 +270,6 @@ def _fetch_metric(self, bucket_name: str, metric: S3Metric) -> float:
             )
         return 0
 
-    def _load_metadata_file(self, bucket_name: str) -> Optional[StorageContainerConfig]:
-        """
-        Load the metadata template file from the root of the bucket, if it exists
-        """
-        if self._is_metadata_file_present(bucket_name=bucket_name):
-            try:
-                logger.info(
-                    f"Found metadata template file at - s3://{bucket_name}/{OPENMETADATA_TEMPLATE_FILE_NAME}"
-                )
-                response_object = self.s3_client.get_object(
-                    Bucket=bucket_name, Key=OPENMETADATA_TEMPLATE_FILE_NAME
-                )
-                content = json.load(response_object["Body"])
-                metadata_config = StorageContainerConfig.parse_obj(content)
-                return metadata_config
-            except Exception as exc:
-                logger.debug(traceback.format_exc())
-                logger.warning(
-                    f"Failed loading metadata file s3://{bucket_name}/{OPENMETADATA_TEMPLATE_FILE_NAME}-{exc}"
-                )
-        return None
-
-    def _is_metadata_file_present(self, bucket_name: str):
-        return self.prefix_exits(
-            bucket_name=bucket_name,
-            prefix=OPENMETADATA_TEMPLATE_FILE_NAME,
-        )
-
```

Comment:
instead of using look before you leap, we will be using easier to ask for forgiveness pattern